Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -692,7 +698,7 @@ public void CanConvertToRaw()
             Assert.Equal(1, rawNext.Header.Index);
             Assert.Equal(1, rawNext.Header.Difficulty);
             Assert.Equal(rawGenesis.Header.Hash.ToArray(), rawNext.Header.PreviousHash.ToArray());
-            Assert.Equal("2018-11-30T00:00:00.000000Z", rawNext.Header.Timestamp);
+            Assert.Equal("2018-11-29T00:00:15.000000Z", rawNext.Header.Timestamp);
```

Comment:
What did you intend behind this change?  Could you elaborate?

Reply:
Recoding #925. we change the threshold to 15s. It makes `BlockHeaderTest.ValidatePreviousHash()` cannot pass on my machine. So I make the block interval to 15 on `TestUtils.MineNext<T>()`
https://github.com/planetarium/libplanet/pull/935/commits/f62fb4f32ab9670f0cca79506b1f3e53e5adff95#diff-a65214bb72ee1228cfc0859975039e53R214